src/components/v1/V1Project/modals/TransferUnclaimedModal.tsx

```diff
@@ -0,0 +1,113 @@
+import { t, Trans } from '@lingui/macro'
+import { Form, Modal } from 'antd'
+import { useWatch } from 'antd/lib/form/Form'
+import InputAccessoryButton from 'components/InputAccessoryButton'
+import { EthAddressInput } from 'components/inputs/EthAddressInput'
+import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { isAddress } from 'ethers/lib/utils'
+import useUnclaimedBalanceOfUser from 'hooks/v1/contractReader/UnclaimedBalanceOfUser'
+import { useTransferTokensTx } from 'hooks/v1/transactor/TransferTokensTx'
+
+import { FC, useContext, useState } from 'react'
+import { formatWad, fromWad, parseWad } from 'utils/format/formatNumber'
+import { tokenSymbolText } from 'utils/tokenSymbolText'
+
+interface TransferUnclaimedModalProps {
+  visible?: boolean
+  onCancel?: VoidFunction
+  onConfirmed?: VoidFunction
+}
+
+export const TransferUnclaimedModal: FC<TransferUnclaimedModalProps> = ({
```

Comment:
This is definitely valid and good thinking, but I think it's probably best if we stick to our current component naming convention:


```
export function TransferUnclaimedModal({
    visible,
    onClose,
    onConfirmed,
}: {
  visible: boolean
  onClose: VoidFunction
  onConfirmed: VoidFunction
}) 
```

**Note:** 
- Since none of these props are optional we don't need/want the `?:` typing 
- `onClose` might be a better name than `onCancel` for closing modals, and is consistent for our modals throughout the app I believe.

Reply:
I agree with you on both points, I've applied the first one as it really doesn't make sense for them to be optional as you said.

However, the second point would keep us inconsistent since all other modals are using `onCancel`, I can make another issue after this PR to rename all of these modals to use `onClose` , wdyt ?

Reply:
Sure, issues for these would be good with the `Maintenance` label. Not huge priorities though I'd put them on the back burner personally

Reply:
Yeah no problem, just wanna address these so we do not forget in future 🍻